bitnami/fluentd/values.yaml

```
@@ -249,6 +249,29 @@ forwarder:
     create: true
     pspEnabled: false
 
+  ## Extra init containers
+  ## Example
+  ##
+  ## extraInitContainers:
+  ##   - name: do-something
+  ##     image: busybox
+  ##     command: ['do', 'something']
+  ##
+  extraInitContainers: []
```

We are standardising the parameter name of `initContainers` instead of `extraInitContainers`
```suggestion
  ## Add init containers to forwarder pods
  ## For example:
  ## initContainers:
  ##   - name: your-image-name
  ##     image: your-image
  ##     imagePullPolicy: Always
  ##
  initContainers:
```